docs/run_test/understand_results.md

```diff
@@ -0,0 +1,73 @@
+# Understand test results
+
+- [Overview](#overview)
+- [Intermediate results](#intermediate-results)
+- [Final results](#final-results)
+
+## Overview
+
+It's essential to understand the results after running tests. LISA has 7 kinds
+of test results in total: 3 of which are intermediate results, and 4 of which
+are final results, as explained in sections below. Each test case can and will
+be moved from one result to another but can never have two or more results at
+the same time. 
+
+![test_results](../img/test_results.png)
```

Comment:
Between the queued and assigned, it should have two lines with opposite direction.

Reply:
The reasons is below, the explanation should be shorter to clear.

queued -> assigned: assign a test case to an environment. (the underlying reason may be to run or to attach deployment failure.
assigned -> queued: environment deployment or initialization is success, no need to use the test case to attach a failure.

Reply:
So, in other words, for queued -> assigned, it means a test in queue gets assigned to an env, either in order to run or to attach (handle?) a deployment failure, and for queued -> assigned, the env gets deployed/initialized successfully. Is my understanding correct?